WalletWasabi.Fluent/Views/Wallets/Home/History/Columns/IndicatorsColumnView.axaml

```diff
@@ -57,16 +57,29 @@
       </c:PrivacyContentControl.Styles>
 
       <PathIcon Opacity="0.6"
-                Classes.NormalTransaction="{Binding IsNormalTransactionDisplayed}"
+                Classes.IncomingTransaction="{Binding IsIncomingTransactionDisplayed}"
+                Classes.OutgoingTransaction="{Binding IsOutgoingTransactionDisplayed}"
+                Classes.SelfTransfer="{Binding IsSelfTransferTransaction}"
                 Classes.Coinjoin="{Binding IsCoinjoinDisplayed}"
                 Classes.Cancellation="{Binding IsCancellationDisplayed}"
                 Classes.CoinjoinGroup="{Binding IsCoinjoinGroupDisplayed}"
                 Classes.CPFP="{Binding IsCPFPDisplayed}">
         <PathIcon.Styles>
           <Style Selector="PathIcon.NormalTransaction">
```

Comment:
All of this style should get removed, it is not needed now.

Reply:
From what I see, it is actually needed; removing these styles would mean that the icon won't display in the view cos that's where they are feeding from.

Also, not checking for null threw an argument null exception error. Giving, for instance, an incoming transaction, the value of the outgoing amount property is null.

Reply:
> From what I see, it is actually needed; removing these styles would mean that the icon won't display in the view cos that's where they are feeding from.

I meant to remove only the first style `NormalTransaction`. I have tested removing it and the icons are displayed just fine.

Reply:
Removed